src/graphql/fragments/user.graphql

```diff
@@ -19,3 +19,10 @@ fragment User on User {
     }
   }
 }
+
+fragment WatchListItem on WatchedColonies {
```

Comment:
Should that belong in this file rather than `fragments/colony`?

Reply:
Yes, that makes sense, moving it to fragments/colony.graphql